examples/teensy4/src/usb_keypress.rs

```diff
@@ -0,0 +1,194 @@
+//! Demonstrates how to send keystrokes via USB
+//! This will spam your keyboard with eldritch slogans until you unplug it or press the reprogram button.
+
+#![no_std]
+#![no_main]
+
+use teensy4_panic as _;
+
+use core::str::Chars;
+use imxrt_usbd::full_speed::BusAdapter;
+use teensy4_bsp::LED;
+use usb_device::device::UsbDevice;
+use usb_device::prelude::{UsbDeviceBuilder, UsbVidPid};
+use usbd_hid::descriptor::KeyboardReport;
+use usbd_hid::descriptor::SerializedDescriptor;
+use usbd_hid::hid_class::HIDClass;
+
+#[cortex_m_rt::entry]
+fn main() -> ! {
+    let support::Peripherals {
+        mut led,
+        mut gpt1,
+        mut ccm,
+    } = support::setup(core::time::Duration::from_millis(500), 115_200);
+
+    //
+    //
```

Comment:
intended to be a comment here?

Reply:
I think I was just separating two sections of code.  

If you decide it would be more illustrative in a different repo, I can turn it into a pull request over there.  I included it here, because there is already a mouse example (which I used as the base).